src/utils/helpers/currencies.ts

```diff
@@ -1,23 +1,7 @@
-import { CurrencyExt, CurrencyIdLiteral } from '@interlay/interbtc-api';
-import {
-  InterBtc, // on Polkadot
-  Interlay, // On Polkadot
-  KBtc, // on Kusama
-  Kintsugi, // On Kusama
-  Kusama, // on Kusama
-  Polkadot // on Polkadot
-} from '@interlay/monetary-js';
+import { CurrencyExt } from '@interlay/interbtc-api';
 
-const CURRENCIES: Array<CurrencyExt> = [Polkadot, InterBtc, Interlay, KBtc, Kintsugi, Kusama];
+// Squid query by currency ticker for native assets and by id for foreign assets.
+// We need to differentiate because those are handled differently on squid side.
+const getCurrencyEqQuery = (currency: CurrencyExt): string => 'id' in currency ? `asset_eq: ${currency.id}` : `token_eq: ${currency.ticker}`
```

Comment:
Don't quite follow this function name - could it be renamed to make it clearer what value is being returned?

Reply:
It's a helper function for squid queries to get correct condition for the currency (as squid handles native and foreign currencies differently so they need different condition),  I changed name of it to `getCurrencyEqualityCondition`, is it easier to understand now? :)

Reply:
Yes that makes a lot more sense :) thank you